recipe/meta.yaml

```diff
@@ -82,10 +82,12 @@ requirements:
     - gsl
     - h5py
     - healpy
+    - healsparse
     - lmfit
     - lsstdesc.coord
     - matplotlib-base
-    - moto
+    - mock =4.0.2
```

Comment:
Have they not fixed this problem yet?

Reply:
resolving the rubin-env version 0.1.4 I still see mock 4.0.3 and moto 1.3.16, which give us the problem.